content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -79,6 +120,21 @@ Then use the functions and data structures in the stub file to develop the serve
 * protocol: UNIX domain socket (`unix`)
 
   The gRPC server should listen at UNIX domain socket.
+{{< /tab >}}
+{{% tab name="v2" %}}
+* kms plugin version: `v2alpha1`
+
+  In response to procedure call Version, a compatible KMS plugin should return `v2alpha1` as `VersionResponse.version`.
```

Comment:
```suggestion
  In response to procedure call Status, a compatible KMS plugin should return `v2alpha1` as `StatusResponse.Version`, "ok" as `StatusResponse.Healthz` and a keyID (KMS KEK ID) as `StatusResponse.KeyID`
```

ref: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go#L238-L255

1. `StatusResponse.Healthz` must be ok. Anything other than `ok` will result in health check failure.
2. `StatusResponse.Version` must be `v2alpha1`.
3. `StatusRespose.KeyID` must not be empty. The recommendation is to return the KMS KEK ID as this will be used for rotation.

If either (1), (2) and (3) are not valid, the health check for the KMS plugin will fail.